doc/sphinx/addendum/universe-polymorphism.rst

```diff
@@ -270,6 +270,44 @@ Insufficiently restrictive variance annotations lead to errors:
 
    Fail Polymorphic Cumulative Record bad@{*a} := {p : Type@{a}}.
 
+.. example:: Demonstration of universe variances
+
+   Here is an example demonstating the behavior of various universe variances.
+
+   .. coqtop:: all
+
+      Set Printing Universes.
+      Set Universe Polymorphism.
+      Set Polymorphic Inductive Cumulativity.
+
+      Inductive Invariant @{=u} : Type@{u}.
+      Inductive Covariant @{+u} : Type@{u}.
+      Inductive Irrelevent@{*u} : Type@{u}.
+
+      Section Universes.
+        Universe u v.
+        Constraint u < v.
+
+        (** An invariant universe blocks cumulativity from upper or lower levels. *)
```

Comment:
```suggestion
        (* An invariant universe blocks cumulativity from upper or lower levels. *)
```

Comments beginning with `(**` are not appearing, can you fix (in 4 places)?

Reply:
3 places and fixed.